doc/api/assert.md

```diff
@@ -319,43 +319,67 @@ are recursively evaluated also by the following rules.
 ```js
 const assert = require('assert').strict;
 
+// This fails because 1 !== '1'.
 assert.deepStrictEqual({ a: 1 }, { a: '1' });
-// AssertionError: { a: 1 } deepStrictEqual { a: '1' }
-// because 1 !== '1' using SameValue comparison
+// AssertionError: Input A expected to strictly deep-equal input B:
```

Comment:
I removed the `ERR_ASSERTION` part in some cases because the line length would otherwise be above 80 characters.